README.md

```diff
@@ -10,22 +10,17 @@ The main entry point into the program is `main.go` in the root directory. You ca
 `main` using `go build main.go` or run it directly using `go run main.go`. With both methods, there are currently
 two subcommands that can be used:
 
+##### Linux / OSX
 1) `./main run [--port=9000]`: This runs the entire backend server. The `-p` or `--port` flag can be
 added to specify the port of the server
 
 2) `./main run <name>`: This creates a new api with the specified name. It modifies all necessary files
 and creates new skeleton code inside the `api/<name>` folder.
 
+##### Windows
+1) If you installed from the package, run the program via powershell with `.\<PROGRAM_NAME>.exe`
```

Comment:
Does the windows version still have subcommands like `run` and `create`?

Reply:
yes